AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/RemoteSyncEngine+AuthModeStrategyDelegate.swift

```diff
@@ -0,0 +1,35 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Amplify
+import Combine
+import Foundation
+import AWSPluginsCore
+
+@available(iOS 13.0, *)
+extension RemoteSyncEngine: AuthModeStrategyDelegate {
```

Comment:
Moved the conformance `RemoteSyncEngine` of `AuthModeStrategyDelegate` to a separate file